how to make a bootable usb.md

```diff
@@ -4,23 +4,13 @@
 
 ### windows
 
-**64 bit**
+**64/32 uefi systems and 64 bios systems**
 
 1 first download the **linuxmint 20 iso [here](https://linuxmint.com/edition.php?id=281)**.
 
 2 then flash the linuxmint iso to your usb with **rufus [here](https://rufus.ie/)**.
 
-3 now download the latest **dahilaosbuild.zip [here](https://github.com/HexaOneOfficial/dahilaos/releases/)**. 
-
-4 then unpack the zip file and than overwrite the files on your linuxmint usb drive. 
-
-**32 bit**
-
-1 first download the **linuxmint 19.3 32bit iso [here](https://www.linuxmint.com/edition.php?id=273/)**.
-
-2 then flash the linuxmint iso to your usb with **rufus [here](https://rufus.ie/)**.
-
-3 now download the latest 32 bit **dahilaosbuild.zip [here](https://github.com/HexaOneOfficial/dahilaos/releases/)**. 
+3 now download the latest **dahilaosbuild.zip [here](https://github.com/HexaOneOfficial/dahliaos/releases)**. 
```

Comment:
*dahlia :)

Reply:
this is the new and correct repo

Reply:
Correct, but I mean the name of the zip file **dahilaosbuild.zip